modules/apps/commerce/commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentEntryDisplayContext.java

```diff
@@ -346,6 +348,24 @@ public String getRelatedToClassPK() {
 		return String.valueOf(_relatedCommercePaymentEntry.getClassPK());
 	}
 
+	public String getRelatedToLink() throws PortalException {
```

Comment:
Lets rename this to getRelatedToURL(), it aligns more with current naming patterns